build.gradle

```diff
@@ -1,13 +1,28 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
+    ext {
+        rootCompileSdk = 31
+        rootMinSdk = 21
+        rootTargetSdk = 31
+        rootVersionCode = 1
+        rootVersionName = "1.0"
+
+        appcompat = '1.3.1'
+        material = '1.4.0'
+        constraintlayout = '2.1.1'
+        fragmentKtx = '1.3.6'
+        viewBindingDelagate = '1.4.7'
+        hilt = '2.38.1'
+    }
+
```

Comment:
<img width="226" alt="image" src="https://user-images.githubusercontent.com/23121846/147235611-b18d3a2a-96b9-4874-ac9e-ed47a2b888aa.png">
How about do something like this? 

This will be more readable + it will be clear that it is version

Reply:
versions.hilt
versions.fragmentKtx and so on